### DISCUSSION TITLE
[Ideas] - Create a Discord Server

### DISCUSSION THREAD

**Discussion by Freebase394** [semaphoreui/semaphore#1410] [Ideas]
Could the dev responsible for this app please create a discord Server so that the whole community can communicate with each other and ask questions faster and more proactively.

I fell here from parachutes and I'm trying to study the app, and I'm having some difficulties in understanding some things!

**Comment by TKILLAHOME** [comment]
I'm onboard with the idea. Having a Discord server would be a fantastic way for us to discuss various topics and enable the community to provide mutual assistance. Plus, it could serve as an excellent platform for addressing Ansible-related matters. Thank you for your great work :)

**Comment by robinmalik** [comment]
Perhaps if someone was willing to create a Discord server, @fiftin would be willing to list it on here and the official website as an unofficial community support server?

**Comment by tboerger** [comment]
I have created a Discord server now, I would give @fiftin owner access if he wish :) https://discord.gg/6AhR7SdE

**Reply by SMAW** [reply]
It's expired... Maybe create a new one?

**Reply by tboerger** [reply]
https://discord.gg/DbkbSvxFrt
